components/ModDashboardLeftNavbar.vue

```diff
@@ -63,4 +67,34 @@ const updateSelectedDashboardView = (event: Event) => {
     const selectedValue = (event.target as HTMLSelectElement).value
     selectedDashboardView.value = selectedValue
 }
+
+interface SubmissionStatusDisplayCount {
+    forReviewCount: number
+    approvedCount: number
+    rejectedCount: number
+}
+
+function createCountsForSubmissionListView(submissions: Submission[]) {
+    const statusCountObject: SubmissionStatusDisplayCount = {
+        forReviewCount: 0,
+        approvedCount: 0,
+        rejectedCount: 0
+    }
+
+    submissions.forEach((submission: Submission) => {
```

Comment:
You can actually use reduce or map for this instead of for each

Reply:
updated to use reduce